cmd/pipe-main.go

```diff
@@ -95,16 +96,19 @@ EXAMPLES:
   3. Copy an ISO image to an object on Amazon S3 cloud storage.
      {{.Prompt}} cat debian-8.2.iso | {{.HelpName}} s3/opensource-isos/gnuos.iso
 
-  4. Stream MySQL database dump to Amazon S3 directly.
+  4. Copy an ISO image to an object on minio storage using s3 encryption.
+     {{.Prompt}} cat debian-8.2.iso | {{.HelpName}} --enc-s3="minio/opensource-isos=my-key-name" minio/opensource-isos/gnuos.iso
```

Comment:
```suggestion
     {{.Prompt}} cat debian-8.2.iso | {{.HelpName}} --enc-s3="minio/opensource-isos=my-key-name" minio/opensource-isos/gnuos.iso
```

You cannot specify any keyname with `enc-s3` - it is `enc-kms`